scripts/backend_tests.py

```diff
@@ -202,7 +202,7 @@ def _convert_to_test_target(path):
                 os.path.join(base_path, root)))
         for subroot, _, files in os.walk(os.path.join(base_path, root)):
             for f in files:
-                if (f.endswith('_test.py') and
+                if (f.endswith('_test.py') or f.endswith('Test.py') and
```

Comment:
Why the addition of Test.py?

If due to third-party stuff, shouldn't that be excluded from the overall set of dirs anyway?